detox/scripts/build_universal_framework_legacy.sh

```diff
@@ -1,76 +0,0 @@
-#!/bin/bash -e
```

Comment:
@asafkorem I wonder, won't removing `legacy` scripts break someone?

Reply:
> Who's our vulnerable audience?

Allegedly, Xcode < 12 if such thing even exist (current Xcode version is 15), though they can update their xcodebuild (CLI) version and they'll be fine.

Xcode 11 is quite outdated. Xcode 12 was released in 2020, and I have no confidence that Detox will compile on such an old version (it also comes with the respective old simctl and xcodebuild command).
IMO there is no good reason to continue maintaining or supporting this script